cli/azd/resources/scaffold/templates/main.bicept

```diff
@@ -82,4 +113,8 @@ output AZURE_RESOURCE_EVENT_HUBS_ID string = resources.outputs.AZURE_RESOURCE_EV
 {{- if .ServiceBus}}
 output AZURE_RESOURCE_SERVICE_BUS_ID string = resources.outputs.AZURE_RESOURCE_SERVICE_BUS_ID
 {{- end}}
+{{- if .AiFoundryProject }}
+output AZURE_AIPROJECT_CONNECTION_STRING string = aiModelsDeploy.outputs.aiFoundryProjectConnectionString
```

Comment:
Would it make sense to add an extra underscore here: `AZURE_AI_PROJECT_CONNECTION_STRING`?